milestones/start_planning.md

```diff
@@ -0,0 +1,97 @@
+Planning is key to a successful event, so lets go through some of the things you'll want to think about.
+
+## Mode of the meetup
+
+There are two main ways to conduct a meetup, Online and In-person.
+
+### Online Meetup
+
+The following is a checklist for having an online meetup
+
+    - Platform to host the meetup
+    - Fix a date and time
+    - Speaker outreach
+    - Social media promotion plans
+    - Platform to have a conversation with the attendees, speakers, and organizers
+
+An online meetup allows you to connect to a broader audience living in different time zones along with the regular meetup group members.  The organizers may want to promote the event on the social media platforms of the time zones coinciding with the event timing to extend the reach of the group.
+
+### In Person meetup
+
+The following is a checklist for having a in person meetup
+
+    - Venue to host the meetup
+    - Fix a date and time for the meetup
+    - Speaker outreach
+    - Sponsorship Scout
+    - Social media promotion plans
+    - Platform to have a conversation with the attendees, speakers, and organizers
+
+
+## Finding speaker
+
+Selecting the topics forms the core of any meetup group. Before looking for speakers, if the organizer pays heed to these points, that will be helpful.
+
+What kind of audience are you expecting in the meetup? What is their experience with Ansible and with Automation in general?
+
+Here are a few place for speaker outreach :
+      1. Social media, email
+      2. Community events
+      3. Github contributions
+      4. Ansible Community Communication platform
+
+While reaching out to the speakers, one has to be careful to refrain from spamming people to give talks at the meetups. If you know the person or someone who can introduce them, that would be wonderful. And if not, a personal touch to the message is necessary for the person to know you, your intention, and your ideas, so they can make a reasoned decision about accepting or declining the invite. Here are some template messages which might be helpful in this process.
```

Comment:
```suggestion
While reaching out to the speakers, be careful to refrain from spamming people to give talks at the meetups. If you know the person or someone who can introduce them, that would be wonderful. And if not, a personal touch to the message is necessary for the person to know you, your intention, and your ideas, so they can make a reasoned decision about accepting or declining the invite. Here are some template messages which might be helpful in this process.
```
I don't see any templates here yet . . .

Reply:
Added this line 'There are some [template messages](./resources/email_templates) in the resource section which might be helpful in this process.'